tracks/x86-64-assembly/exercises/two-fer/mentoring.md

```diff
@@ -0,0 +1,94 @@
+### Reasonable solutions
+
+Using a hand-rolled loop:
```

Comment:
You might want to consider using heading at level 4 here. It's no requirement, but might look better in some views :)

```suggestion
#### Hand-rolled loop
```